content/en/database_monitoring/setup_sql_server/troubleshooting.md

```diff
@@ -120,9 +120,10 @@ If you are using the latest version of the Microsoft OLE DB Driver for SQL Serve
 
 1. If you are using a self-signed certificate and the Force Encryption setting on the server (`rds.force_ssl=1` on AWS) to ensure clients connect with encryption:
 
-   - Change to a certificate that is trusted as part of the client's trust chain
-   - Add the self-signed certificate as a trusted certificate on the client
-   - Add `TrustServerCertificate=yes;` to the connection string
+   - Change to a certificate that is trusted as part of the client's trust chain.
+   - Add the self-signed certificate as a trusted certificate on the client.
+   - If you're using Microsoft OLE DB Driver 19, add `Trust Server Certificate=True` to the connection string, including the spaces.
```

Comment:
This is nested under a paragraph above that specifies `If you are using the latest version of the Microsoft OLE DB Driver for SQL Server...` and I believe it is duplicated below, starting at L152.

Reply:
@EmmaOnThursday Thanks for catching that -- I've pushed a change that organizes the content by version. Is this correct? I was struggling to parse which steps applied to which section. Thank you!